src/webextension/manage/actions.js

```diff
@@ -83,6 +86,7 @@ function addItemStarting(actionId, item) {
 }
 
 function addItemCompleted(actionId, item) {
+  telemetry.recordEvent("lockbox", "itemAdded", "addItemForm", {"itemid": item.id});
```

Comment:
This isn't the right place to put this; it should go above in `addItem`. This function can run when *another page* has added an item and the current page is just syncing its local cache. If you had multiple pages open (which, granted, is going away temporarily* in #164), this event would be logged multiple times for a single item.

\* I say "temporarily" because we'll have the possibility of multiple pages again when we have the popup reader UI.